app/service/CasesService.scala

```diff
@@ -416,6 +420,15 @@ class CasesService @Inject()(appConfig: AppConfig,
     addEvent(original, updated, details, operator)
   }
 
+  private def addCaseCreatedEvent(liabilityCase: Case, operator: Operator)
+                                 (implicit hc: HeaderCarrier): Future[Unit] = {
+    val event = NewEventRequest(CaseCreated("Liability case created"), operator)
+    connector.createEvent(liabilityCase, event) recover {
+      case t: Throwable =>
+        Logger.error(s"Could not create Event for case [${liabilityCase.reference}] with payload [$event]", t)
+    } map (_ => ())
```

Comment:
I think we could reuse the existing `addEvent` here
```suggestion
    val details = NewEventRequest(CaseCreated("Liability case created"), operator)
    addEvent(liabilityCase, liabilityCase, details, operator)
```

Reply:
We didn't use that one because that takes the original and the updated case but on our scenario we are only just creating the case so have only 1 case object hence we created the addCaseCreatedEvent that takes only the case. However I suppose we can use the addEvent method and pass the case as a original and updated?